common/lib/xmodule/xmodule/templates/about/overview.yaml

```diff
@@ -42,7 +42,7 @@ data: |
         <article class="response">
           <h3>What web browser should I use?</h3>
           <p>The Open edX platform works best with current versions of Chrome, Firefox or Safari, or with Internet Explorer version 9 and above.</p>
-          <p>See our <a href="http://edx.readthedocs.org/en/latest/browsers.html">list of supported browsers</a> for the most up-to-date information.</p>
+          <p>See our <a href="http://edx.readthedocs.org/projects/docstrings/en/latest/front_matter/browsers.html">list of supported browsers</a> for the most up-to-date information.</p>
```

Comment:
thanks so much for catching this error!
As it is, the new URL points to the browser topic in the Research Guide. I suggest, instead, that it go to the same topic in the Open edX Learner's guide, at 
http://edx.readthedocs.org/projects/open-edx-learner-guide/en/latest/front_matter/browsers.html
Once that guide is open, there might be other information of value to those reading these FAQ-type pointers.